controllers/eventbus/installer/nats.go

```diff
@@ -619,10 +619,10 @@ func (i *natsInstaller) buildStatefulSetSpec(serviceName, configmapName, authSec
 	if err != nil {
 		return nil, fmt.Errorf("failed to get nats streaming version, err: %w", err)
 	}
-	// Streaming requires minimal size 3.
+	//
 	replicas := i.eventBus.Spec.NATS.Native.Replicas
 	if replicas < 3 {
-		replicas = 3
+		i.logger.Errorw("streaming recommends a replica >= 3, '%v'", replicas)
```

Comment:
Use `Warnw`, and make the log message a complete sentence.

Reply:
I reverted this back instead of fixing it since it was in the same file as above with the STAN implementation